apps/judicial-system/web/src/routes/Court/components/CourtCaseNumber/CourtCaseNumberInput.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { Box, Button, Input } from '@island.is/island-ui/core'
+import { isIndictmentCase } from '@island.is/judicial-system/types'
+import { BlueBox } from '@island.is/judicial-system-web/src/components'
+import { CaseState } from '@island.is/judicial-system-web/src/graphql/schema'
+import { TempCase as Case } from '@island.is/judicial-system-web/src/types'
+import {
+  removeTabsValidateAndSet,
+  validateAndSendToServer,
+} from '@island.is/judicial-system-web/src/utils/formHelper'
+import {
+  UpdateCase,
+  useCase,
+} from '@island.is/judicial-system-web/src/utils/hooks'
+import { validate } from '@island.is/judicial-system-web/src/utils/validate'
+
+import { courtCaseNumber } from './CourtCaseNumber.strings'
+import * as styles from './CourtCaseNumber.css'
+
+interface Props {
+  workingCase: Case
+  setWorkingCase: React.Dispatch<React.SetStateAction<Case>>
+}
+
+const CourtCaseNumberInput: React.FC<React.PropsWithChildren<Props>> = (
```

Comment:
Please consider importing FC from react and also removing PropsWithChildren, as this component does not use Children